documentation/modules/auxiliary/scanner/http/scraper.md

```diff
@@ -7,15 +7,22 @@ This module scrapes data from a specific web page based on a regular expression.
 2. Do: ```set RHOSTS [IP]```
 3. Do: ```run```
 
+## Options
+
+### PATH 
+
+The path from where the data is to be scraped from.
+ 
+### PATTERN
+
+The regular expression to use . Default value:`<title>(.*)</title>` which simply grabs page title.
```

Comment:
grabs **the** page title

Reply:
A regular expression to capture data from the webpage.

Reply:
Done.